Batteries/Linter/TrailingWhitespace.lean

```diff
@@ -0,0 +1,48 @@
+import Lean.Elab.Command
+import Lean.Linter.Util
+
+/-!
+#  The "trailingWhitespace" linter
+
+The "trailingWhitespace" linter emits a warning whenever a line ends with a space or a file
+does not end with a line break.
+-/
+
+open Lean Elab
+
+namespace Batteries.Linter
+
+/--
+The "trailingWhitespace" linter emits a warning whenever a line ends with a space or a file
+does not end with a line break.
+-/
+register_option linter.trailingWhitespace : Bool := {
+  defValue := true
```

Comment:
```suggestion
  defValue := false
```

Reply:
As suggested by Mario.  There would still be the need to switch the option to true when building Batteries.